rllib/offline/mixed_input.py

```diff
@@ -1,11 +1,13 @@
-import numpy as np
+from types import FunctionType
+from typing import Dict
 
+import numpy as np
 from ray.rllib.offline.input_reader import InputReader
-from ray.rllib.offline.json_reader import JsonReader
 from ray.rllib.offline.io_context import IOContext
+from ray.rllib.offline.json_reader import JsonReader
 from ray.rllib.utils.annotations import override, DeveloperAPI
 from ray.rllib.utils.typing import SampleBatchType
-from typing import Dict
+from ray.tune.registry import RLLIB_INPUT, _global_registry
```

Comment:
Hmm, is it possible to avoid importing the RLLIB_INPUT and _global_registry here? I'm thinking perhaps we can have some helper method